src/components/example/certificate/CertificateForm.tsx

```diff
@@ -151,6 +197,26 @@ const CertificateForm: FC<{ initialValues: ICertificate }> = ({
             min={formatDateToYYYYMMDD(new Date(formValues?.validFrom) as Date)}
             onChangeValue={handleInputChange}
           />
+          <div className="form-input">
+            <label htmlFor="">{translate('Assigned users')}</label>
+            <Button
+              type="button"
+              className="add-participant-button"
+              onClick={() => setIsUserLookup(true)}
+              icon={<img width={20} height={20} src={SearchIcon} />}
+              label={translate('Add Participant')}
+            />
+          </div>
+          <TableComponent
+            data={
+              assignedUsers.length > 0
+                ? assignedUsers
+                : formValues?.assignedUsers || []
+            }
+            columns={columns}
+            renderActions={(id) => renderActions(id)}
+            className="search-table"
+          />
```

Comment:
### Recommendation
- **Data Prop Logic:**
   - The ternary operator for the `data` prop is a bit complex. Consider simplifying or extracting this logic.
   - There's potential for inconsistency between `assignedUsers` and `formValues?.assignedUsers`. Ensure these are properly synchronized.
- **EnderActions Prop:**
The renderActions prop is passing a function that simply calls another function. This can be simplified.

```JSX
const tableData = useMemo(() => {
    return assignedUsers.length > 0 ? assignedUsers : formValues?.assignedUsers || [];
  }, [assignedUsers, formValues?.assignedUsers]);

  return (
    <TableComponent
      data={tableData}
      columns={columns}
      renderActions={renderActions}
      className="search-table"
    />
```